libs/ledgerjs/packages/hw-app-cosmos/src/Cosmos.ts

```diff
@@ -79,7 +79,7 @@ export default class Cosmos {
   }
 
   serializeHRP(hrp: string): Buffer {
-    if (hrp == null || hrp.length < 3 || hrp.length > 83) {
+    if (hrp == null || hrp.length === 0 || hrp.length > 83) {
```

Comment:
Do you remember what pb this fixed?
Ideally we need to test and non-reg that too...

Reply:
Yes, the nyx prefix is "n"